package.json

```diff
@@ -28,8 +28,7 @@
     "gulp-concat": "^2.5.2",
     "gulp-uglify": "^1.1.0",
     "markdown": "*",
-    "showdown": "*",
-    "xregexp": "3.2.0"
+    "showdown": "*"
```

Comment:
unrelated to this: wouldn't it be better to pin down the version?

Reply:
I think the upstream devs don't actually care about the version of those. I'm pretty sure the reason that they're included is for benchmarking differences in speed, so they probably wanted the latest version to compare against.

Also, even though this fork is horribly out of date with upstream, I don't think that's something we need to be changing